datatable/java/datatable/src/main/java/io/cucumber/datatable/DataTable.java

```diff
@@ -620,26 +620,32 @@ public String toString() {
     /**
      * Prints a string representation of this
      * table to the {@code appendable}.
+     * 
+     * @deprecated superseded by TablePrinter.printTable()
```

Comment:
You can use `@link` to link to a method in Java doc.

```suggestion
     * @deprecated superseded by {@link TablePrinter#printTable(DataTable, Appendable)}
```